ambrs/aerosol.py

```diff
@@ -67,6 +67,7 @@ class AerosolModeDistribution:
     species: tuple[AerosolSpecies, ...]
     number: RVFrozenDistribution                     # modal number concentration distribution
     geom_mean_diam: RVFrozenDistribution             # geometric mean diameter distribution
+    log10_geom_std_dev: float                        # mode-specific logarithmic diameter std dev
```

Comment:
looks good! later, this could be something we sample (e.g., RVFrozenDistribution type)